lib/chef_zero/endpoints/organization_user_endpoint.rb

```diff
@@ -20,29 +20,7 @@ def delete(request)
         json_response(200, ChefData::DataNormalizer.normalize_user(user, request.rest_path[3], ['username'], server.options[:osc_compat]))
       end
 
-      def post(request)
```

Comment:
Rather than removing this I think we need to move it and slightly change this code.  The actual endpoint that does this is:

POST /organizations/ORGNAME/users

with a body of `{"username": "USERTOADD"}`